client/src/document/on-github.tsx

```diff
@@ -62,7 +63,10 @@ MDN URL: https://developer.mozilla.org$PATHNAME
 * Folder: \`$FOLDER\`
 * MDN URL: https://developer.mozilla.org$PATHNAME
 * GitHub URL: $GITHUB_URL
+* Last commit hash: \`$HASH\`
 * Document last modified: $DATE
+* Branch: \`$BRANCH\`
+* Repository: \`$REPOSITORY\`
```

Comment:
This will come in somewhat handy when we have translated content because it'll be something like `mdn/translated-content` or `mdn/translated-content-ja`. 
But is it really useful? The `GitHub URL` is super helpful and that will already have the repository information in it. 
I suggest we remove this line.